src/content/research/agent-web-and-its-interface.md

```diff
@@ -3,6 +3,7 @@ title: "Navigating the Future: AX, the Agent Web, and Its Interface"
 description: "In order for website and agents to coexist within a mutually beneficial web, we have to provide a different view of the web with optimal AX and UX without the negative impacts on websites today."
 pubDate: 2025-02-05
 author: "Sean Roberts"
+authorTitle: "Head of AX Architecture, Netlify"
```

Comment:
New, optional – added to enable a bit more meta information at the top of the article: 
<img width="1326" alt="image" src="https://github.com/user-attachments/assets/cbdbc326-0d5c-497f-9ffb-52329af40826" />